lambdas/service/app.py

```diff
@@ -1743,7 +1743,12 @@ def validate_version(version: str) -> None:
                     requestIndex=int,
                     replica=validate_replica,
                     drsUri=str,
-                    token=str)
+                    token=str,
+                    # When Azul generates retry URLs, additional parameters are
+                    # sourced from :meth:`File.to_request`. File subclasses add
+                    # additional parameters that are plugin-specific and thus
+                    # difficult to validate.
```

Comment:
```suggestion
                    # difficult to validate here.
```

Where are these additional parameters validated and do we return a 4xx error as we should if one of them is invalid?